index.html

```diff
@@ -71,7 +124,8 @@
 $(document).ready(function () {
     $('select').material_select();
     //$('ul.tabs').tabs();
-    $('.github-card').html('<img src="http://placehold.it/100x100" />')
+    $('.github-card').html('<img src="http://placehold.it/100x100" />');
+    $('#current-year').html(new Date().getFullYear());
```

Comment:
this should be moved above `$('.github-card').html(...` to avoid modifying that line.

And again, probably this needs to be fixed in a linter issue, using a bear we already have.

Reply:
> this needs to be fixed in a linter issue, using a bear we already have.

Via a JS Bear? or a HTML bear?

Reply:
Whichever you believe is best for the job

Reply:
OK 👍  I will look for it! Also found another helpful linter `JS Standard`